src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -28,274 +43,316 @@ function setupTest(
-          )
-        );
-
-        test(
-          'drawers focus toggles between open and close buttons',
-          setupTest(
-            async page => {
-              const triggerSelector = wrapper.findDrawerTriggerById('pro-help').toSelector();
-              await page.click(triggerSelector);
-              await page.keys('Enter');
-              await expect(page.isFocused(triggerSelector)).resolves.toBe(true);
-              await page.keys('Enter');
-              await expect(page.isFocused(wrapper.findActiveDrawerCloseButton().toSelector())).resolves.toBe(true);
-              await page.keys('Enter');
-              await expect(page.isFocused(triggerSelector)).resolves.toBe(true);
-            },
-            { pageName: 'with-drawers', visualRefresh, mobile }
-          )
-        );
-
-        test(
-          'focuses tools panel closed button when it is opened using keyboard and caused split panel to change position',
-          setupTest(
-            async page => {
-              await page.setWindowSize({ width: 1000, height: 800 });
-              await page.click(wrapper.findSplitPanel().findOpenButton().toSelector());
-              await page.keys(['Tab', 'Tab', 'Tab', 'Tab', 'Enter']);
-              await expect(page.isFocused(wrapper.findToolsClose().toSelector())).resolves.toBe(true);
-            },
-            { pageName: 'with-split-panel', visualRefresh, mobile, splitPanelPosition: 'side' }
-          )
-        );
-
-        test(
-          'focuses split panel preferences button when its position changes from bottom to side',
-          setupTest(
-            async page => {
-              await page.click(wrapper.findSplitPanel().findOpenButton().toSelector());
-              await page.click(wrapper.findSplitPanel().findPreferencesButton().toSelector());
-              await page.keys(['Tab', 'Right', 'Tab', 'Tab', 'Enter']);
-              await expect(page.isFocused(wrapper.findSplitPanel().findPreferencesButton().toSelector())).resolves.toBe(
-                true
-              );
-            },
-            { pageName: 'with-split-panel', visualRefresh, mobile, splitPanelPosition: 'bottom' }
-          )
-        );
-
-        test(
-          'focuses split panel preferences button when its position changes from side to bottom',
-          setupTest(
-            async page => {
-              await page.click(wrapper.findSplitPanel().findOpenButton().toSelector());
-              await page.click(wrapper.findSplitPanel().findPreferencesButton().toSelector());
-              await page.keys(['Tab', 'Left', 'Tab', 'Tab', 'Enter']);
-              await expect(page.isFocused(wrapper.findSplitPanel().findPreferencesButton().toSelector())).resolves.toBe(
-                true
-              );
-            },
-            { pageName: 'with-split-panel', visualRefresh, mobile, splitPanelPosition: 'side' }
-          )
-        );
+        [true, false].forEach(appLayoutWidget => {
```

Comment:
It seems like it is going to do a redundant testing for classic+toolbar feature flag combination.

How about flattening it in this list: https://github.com/cloudscape-design/components/blob/1b116cef645c93b692a4aaf63cb36360d954a32a/src/app-layout/__integ__/app-layout-split-panel.test.ts#L91

Bonus points for creating `describeEachAppLayout` for integration tests (similar to our [unit tests helper](https://github.com/cloudscape-design/components/blob/main/src/app-layout/__tests__/utils.tsx#L64), but that's for future follow ups)

Reply:
Like this way better. Thanks for suggesting